### PR TITLE
More kt

### PR DESCRIPTION
**Pull request by SooStrator1136**
You should give me repo access or smth, making pr's is kinda annoying + create a dev branch or smth

### CODE REVIEW

File: src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import com.paragon.api.module.Category
+import com.paragon.api.module.Module
+import com.paragon.api.setting.Setting
+import com.paragon.api.util.entity.EntityUtil
+import com.paragon.api.util.player.PlayerUtil
+import com.paragon.api.util.render.ColourUtil
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.entity.player.EntityPlayer
+import net.minecraft.util.math.Vec3d
+import org.lwjgl.opengl.GL11
```
Comment (Wolfsurge) [inline src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:12]: Use static imports

idk what they're called in kt

File: src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    private val firstPerson = Setting("FirstPerson", false)
+        .setDescription("Render the hat in first person")
+
+    private val others = Setting("Others", true)
+        .setDescription("Render the hat on other players")
+
+    override fun onRender3D() {
+        mc.world.playerEntities.forEach {
+            // We don't want to render the hat
+            if (it === mc.player && !firstPerson.getValue() && mc.gameSettings.thirdPersonView == 0 || !others.getValue() && it !== mc.player) return
```
Comment (Wolfsurge) [inline src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:39]: Don't use inline returns (code style)

File: src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        val vec = EntityUtil.getInterpolatedPosition(player).add(
+            Vec3d(
+                -mc.renderManager.viewerPosX,
+                -mc.renderManager.viewerPosY + player.getEyeHeight() + 0.5 + if (player.isSneaking) -0.2 else 0.0,
+                -mc.renderManager.viewerPosZ
+            )
+        )
+
+        // Change vec if elytra flying
+        if (player.isElytraFlying) vec.add(Vec3d(PlayerUtil.forward(2.0)[0], -0.8, PlayerUtil.forward(2.0)[2]))
```
Comment (Wolfsurge) [inline src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:66]: no inline ifs

File: src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt (hunk 1 above)
Comment (Zane2b2t) [reply src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:12]: Import.your.mom

Comment (Wolfsurge) [reply src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:12]: die

**Comment by SooStrator1136** [comment]
Uhh ill do all these things later

File: src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.entity.player.EntityPlayer
+import net.minecraft.util.math.Vec3d
+import org.lwjgl.opengl.GL11
+import java.awt.Color
+import kotlin.math.cos
+import kotlin.math.sin
+
+/**
+ * @author SooStrator1136
```
[the remark this answers is not in the document]
Comment (SooStrator1136) [reply src/main/java/com/paragon/client/systems/module/impl/render/ChinaHat.kt:18]: Oh yeah

**Pull request merged**